src/APIM_ARMTemplate/apimtemplate/Common/Templates/Logger/LoggerTemplateUtils.cs

```diff
@@ -0,0 +1,45 @@
+﻿using apimtemplate.Common.Constants;
+using apimtemplate.Common.Helpers;
+using apimtemplate.Common.Templates.Abstractions;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace apimtemplate.Common.Templates.Logger
+{
+
+    internal static class LoggerTemplateUtils
```

Comment:
I think it's best not to use abbreviations in code, but that's just a personal preference

Reply:
I dont like them either, the thing is that all code that was placed in `LoggerExtractor` and other `<...>Extractor` classes had some utility methods like these. That is why I have moved them in other place to have static methods separate from API methods of specific extractor.

I have left it as `Utils` because there are a couple of other `Utils` classes just to maintain some style for a while. I will definitely change it in future